### PR TITLE
backup.get should return the latest sandbox backup

### PR DESCRIPTION
**Pull request by LavrovArtem**
@churkin @inikulin @VasilyStrelyaev

### CODE REVIEW

File: test/client/fixtures/sandbox/backup-test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    iframeSandbox.on(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, iframeSandbox.iframeReadyToInitHandler);
+});
+
+QUnit.testDone(function () {
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+});
+
+if (browserUtils.isIE) {
+    test('backup.get should be return actually last created sandbox backup', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/backup-test.js]:
```
    test('backup.get should return the latest sandbox backup', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

File: test/client/fixtures/sandbox/backup-test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        var iframe = document.createElement('iframe');
+
+        iframe.id = 'test';
+        document.body.appendChild(iframe);
+
+        iframe.contentDocument.open();
+        iframe.contentDocument.write('Hello!');
+        iframe.contentDocument.close();
+
+        ok(backup.get(iframe.contentWindow) === iframe.contentWindow['%hammerhead%'].sandbox);
```
Comment (churkin) [inline test/client/fixtures/sandbox/backup-test.js]: use `strictEqual`

**Comment by LavrovArtem** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
ping @inikulin

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
@LavrovArtem Is there any issue related to this PR?

**Comment by LavrovArtem** [comment]
@inikulin In PR #480 fails tests

**Pull request merged**